### ISSUE TITLE
/buy :x: Not a valid item name.

### ISSUE THREAD

**Issue by Maxhem2** [thejoabo/autofishbot#86]
sometimes when all options are on in config it makes me say this title

**Comment by thejoabo** [comment]
Hello @Maxhem2, can you provide screenshots containing the command sent to trigger this response, please ?

**Comment by Maxhem2** [comment]
![image](https://user-images.githubusercontent.com/95474622/219879307-b710af6c-9b2d-46a2-a99f-250a2cb53c6a.png)
This is all I can see. But it does not seem to auto buy better rods (I started today and had 479k and it still used the first rod)

**Comment by thejoabo** [comment]
@Maxhem2 can you send a screenshot from your config file here ? My bot doesn't buy new rods automatically. Also, try to click the "/buy" command (highlighted in blue) to see if it shows exactly the prompt issued to the server.

**Comment by Maxhem2** [comment]
/buy item: rod
[SYSTEM]
user_token = ***
user_cooldown = 3.5
channel_id = ***
debug = False

[CAPTCHA]
ocr_api_key = ***

[NETWORK]
user_agent = 
proxy_ip = 
proxy_port = 
proxy_auth_user = 
proxy_auth_password = 

[AUTOMATION]
boosts_length = 5
more_fish = True
more_treasures = True
fish_on_exit = True
auto_daily = True
auto_buy_baits = True
auto_sell = True
auto_update_inventory = True

[MENU]
compact_mode = False
refresh_rate = 0.3

[COSMETIC]
pet = dolphin
bait = fish
biome = ocean

**Comment by thejoabo** [comment]
That's really confusing. 

> /buy item: rod

This shouldn't happen in any case. Never. It might be related to some change in the JSON event from the command list. I'll need to investigate this further to understand how it occurs.

**Comment by peterProgramer** [comment]
yes it happened because u need to upcase the first letter so it Fish not fish
